database/resources/official-docs.json

```diff
@@ -54,5 +54,12 @@
     "url": "https://nuxtjs.org/",
     "category": "resources",
     "subcategory": "officialdocs"
+  },
+  {
+    "name": "Node.js",
+    "description": "Node.js is built on top of the Google Chrome V8 JavaScript engine, and it's mainly used to create web servers - but it's not limited to just that.",
```

Comment:
```suggestion
    "description": "Node.js is built on top of the Google Chrome V8 JavaScript engine, and it's mainly used to create web servers - but it can be used for other purposes too.",
```